assets/scss/components/_buttonNeue.scss

```diff
@@ -0,0 +1,323 @@
+$C_accent: rgba(241,58,89,1);
```

Comment:
No, the design team picked a new (only slightly different) red. [PR here](https://github.com/meetup/meetup-swatches/pull/54)